### ISSUE TITLE
Problems after Attribute Deletion

### ISSUE THREAD

**Issue by nhp** [avstudnitz/AvS_FastSimpleImport#199]
If you have configured fastsimpleimport to create new options for selected attributes and you then delete the attribute it causes some problems. 
In the selectbox the attribute disappears, so it is not unselectable (probably because it is not present in the system) but it is still in the core_config_data value so during imports it is causing problems due to ```'Mage_Core_Exception' with message 'Attribute XXX not found'```. 
Is there any possibility to ignore attributes from this list that are not present or even better to remove deleted attributes from the list?
As far as I investigated there is no event that is usable, at least if you remove the attributes from a upgrade-script.

**Comment by paales** [comment]
@nhp Yeah, thought about ignoring the attributes when they aren't found, but haven't got time to implement it. Care to make pull request (dev branch)?

**Comment by nhp** [comment]
Probably not until next week but then i will do so.

**Comment by paales** [comment]
:+1:

**Comment by paales** [comment]
@nhp Next week has passed ;)
